pkg/sql/sem/builtins/builtins.go

```diff
@@ -1868,7 +1868,7 @@ var builtins = map[string]builtinDefinition{
 			Fn: func(_ *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
 				return tree.NewDFloat(tree.DFloat(rand.Float64())), nil
 			},
-			Info:       "Returns a random float between 0 and 1.",
+			Info:       "Returns a random floating-point number between 0 and 1. Note that the value contains at most 53 bits of randomness.",
```

Comment:
while we're here, wanna say whether this is inclusive or exclusive?

Reply:
Good idea. Done.